js/main.js

```diff
@@ -25,6 +23,90 @@ $form.addEventListener('submit', event => {
   data.entries.unshift(entry);
   data.nextEntryId++;
 
+  ul.prepend(renderEntry(entry));
+  viewSwap('entries');
+  toggleNoEntries();
+
   $photoPreview.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
 });
+
+// Render Entry Function
+function renderEntry(entry) {
+  const li = document.createElement('li');
+
+  const rowDiv = document.createElement('div');
+  rowDiv.className = 'row';
+  li.appendChild(rowDiv);
+
+  const imgContainerColumnHalfDiv = document.createElement('div');
+  imgContainerColumnHalfDiv.className = 'img-container column-half';
+  rowDiv.appendChild(imgContainerColumnHalfDiv);
+
+  const img = document.createElement('img');
+  img.setAttribute('src', entry.URL);
+  imgContainerColumnHalfDiv.appendChild(img);
+
+  const columnHalfDiv = document.createElement('div');
+  columnHalfDiv.className = 'column-half';
+  rowDiv.appendChild(columnHalfDiv);
+
+  const h3 = document.createElement('h3');
+  h3.textContent = entry.title;
+  columnHalfDiv.appendChild(h3);
+
+  const p = document.createElement('p');
```

Comment:
Don't forget to include the `$` before the variable names. We are creating DOM elements, so we should make sure to include that `$`!